cmake/Modules/ObsDefaults_macOS.cmake

```diff
@@ -0,0 +1,144 @@
+cmake_minimum_required(VERSION 3.20)
```

Comment:
The root CMakeLists.txt changed the minimum to 3.16...3.21. Should this also be changed?

Reply:
Replied elsewhere that leaving this as-is is fine.